chemked/chemked_schema.yaml

```diff
@@ -1,8 +1,9 @@
 # Common reference for authors' information
-author:
+author: &author
```

Comment:
The `author` field should be required. I also found that it didn't work properly when I put the reference here, but it passes the tests, so... who know!